src/en/guide/introduction/whatsNew.adoc

```diff
@@ -1,13 +1,40 @@
-This section covers all the new features introduced in Grails 6
+This section covers all the new features introduced in Grails 7
 
 === Overview
 
-Grails framework 6 updates Spring Boot to version 2.7. We recommend checking the following Spring technologies release notes for more information.
+Grails framework 7 updates Spring Boot to version 3.4. We recommend checking the following Spring technologies release notes for more information.
 
-* https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes[Spring Boot 2.7]
+* https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.4-Release-Notes[Spring Boot 3.4]
 
-The minimum Java version required to run Grails 6 has been updated to Java 11.
```

Comment:
I think there are topics in https://github.com/grails/grails-core/blob/7.0.x/UPGRADE7.md that have not made it into the whatsNew document - see the Cool New Features.  Should we add them here?

Reply:
100% yes.